Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -993,6 +993,18 @@ class RunWorldLeaksTests(RunWebKitTests):
     description = ["world-leaks-tests running"]
     descriptionDone = ["world-leaks-tests"]
     resultDirectory = os.path.join(RunWebKitTests.resultDirectory, "world-leaks-layout-test-results")
+    command = ["python3", "Tools/Scripts/run-webkit-tests",
+               "--no-build",
+               "--no-show-results",
+               "--no-new-test-results",
+               "--clobber-old-results",
+               "--builder-name", WithProperties("%(buildername)s"),
```

Comment:
We should strip out `--builder-name`, `--build-number`, etc. These are only for reporting.

Reply:
Stripping them out now!